hangman.py

```diff
@@ -0,0 +1,222 @@
+# Problem Set 2, hangman.py
+# Name: Sinelnyk Glib
+# Collaborators: -
+# Time spent:
+
+# Hangman Game
+# -----------------------------------
+
+import random
+import string
+
+WORDLIST_FILENAME = "words.txt"
+VOWELS = {"a", "e", "o", "i", "u"}
+HINTS = "*"
+LOWOER_SPACE = "_"
```

Comment:
It`s better to name your constant due to it meaning (here - unknown word), but not due to it value